### ISSUE TITLE
ATAG & Accessibility

### ISSUE THREAD

**Issue by mgifford** [canada-ca/digital-playbook-guide-numerique#212]
Worth specifically mentioning ATAG 2.0 in "For authoring tools such as content management systems (CMS), blog software, and WYSIWYG editors, follow Authoring Tool Accessibility Guidelines (ATAG)."

ATAG 2.0 is in 2 parts. It should be clear that you want both parts addressed. Most stumble over Part A and barely look at Part B. 

Also useful to highlight those systems that have already started addressing this such as Drupal.
  https://www.drupal.org/project/issues/search?issue_tags=atag

This should also be held up as a great example of a standard that is Built in Canada:
  https://www.w3.org/TR/ATAG20/

 Jutta Treviranus & Jan Richards of the Inclusive Design Institute at OCAD University really lead this. 

This should also be seen as a cost savings issue for both accommodation & post-publishing remediation is the most expensive way to address the system.

**Comment by pjackson28** [comment]
@Jeromefb Please see the suggestion regarding ATAG 2.0 (for Standard 2).

**Comment by JuliannaR** [comment]
WCAG, ATAG and UAAG should all be considered when looking at standard requirements for accessibility.
